lib/transaction_builder.js

```diff
@@ -314,6 +314,27 @@ module.exports = class TransactionBuilder {
     return this
   }
 
+  /**
+   * Set the Txn's originSignature, method called from hardware_libs
+   * @param {String | Uint8Array} to hardwareOrigin Signature (hexadecimal)
+   */
+    setOriginSign(hardwareOriginSign) {
```

Comment:
I think you rename to signature, as it may or not be hardware based, depending on the usage from the client.

Reply:
Sure, no worries